src/include/miopen/layernorm.hpp

```diff
@@ -23,8 +23,8 @@
  * SOFTWARE.
  *
  *******************************************************************************/
+#define MIOPEN_BETA_API 1
```

Comment:
This should not be required since its defined in CMake.

Reply:
@JehandadKhan Right! Thanks for informing. We do not need this in library sources and in tests.

Reply:
fixed it.